nf_research_tools.rdb.model.csv

```diff
@@ -55,15 +55,15 @@ Tumor Type,"The types of tumors that are banked, if applicable.","cutaneous neur
 Disease Type,"The diseases of the banked specimens, if applicable.","Neurofibromatosis type 1, Neurofibromatosis type 2, Schwannomatosis",,TRUE,,Biobank,,,list
 Specimen Preparation Method,The preservation methods used by the biobank.,"Fresh collected, Flash frozen, FFPE, Cryopreserved, OCT, RNA later, Formalin-fixed, Ethanol",,TRUE,,Biobank,,,list
 Specimen Format,How the specimens have been processed in preparation for distribution.,"whole tumor, slide, whole cell lysate, DNA, RNA",,TRUE,,Biobank,,,list
-Observation,"A remark, statement, or comment based on the resource.",,"Component, Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Observation Text, Observation Type, Observation Reference, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link",FALSE,,Observation,"Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Observation Text, Observation Type, Observation Reference, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link","Resource, Investigator, Publication",
+Observation,"A remark, statement, or comment based on the resource.",,"Component, Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Species, Observation Text, Observation Type, Observation Phase, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link",FALSE,,Observation,"Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Species, Observation Text, Observation Type, Observation Phase, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link","Resource, Investigator, Publication",

```

Comment:
```suggestion
Observation,"A remark, statement, or comment based on the resource.",,"Component, Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Observation Text, Observation Type, Observation Phase, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link",FALSE,,Observation,"Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Observation Text, Observation Type, Observation Phase, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link","Resource, Investigator, Publication",

```

Reply:
No need to add species to this table if we can get it from another table (e.g Donor) via joins :)

Reply:
> No need to add species to this table if we can get it from another table (e.g Donor) via joins :)

got it! this makes sense now that I just did added the join in the synapse table